models/request_entities.go

```diff
@@ -45,3 +45,9 @@ type Resolution struct {
 	DateOfResolution string   `json:"date_of_resolution" validate:"required,datetime=2006-01-02"`
 	Attachments      []string `json:"attachments" validate:"required"`
 }
+
+// StatementOfAffairs is the model to represent a resolution for an insolvency case
+type StatementOfAffairs struct {
```

Comment:
We should probably add this to the Insolvency response struct as well, so that if we need to query the entire Insolvency object from the database (to send to CHIPS, for example), then we also have the SOA data being returned as part of that Insolvency response